src/filters/pr_executor_pass.hpp

```diff
@@ -11,11 +16,69 @@
 namespace mesos {
 namespace serenity {
 
+inline std::list<ResourceUsage_Executor> filterPrExecutors(
```

Comment:
filterPrExecutors - does it filters out PR executors? :) I guess not.
How about 
```
static ResourceUsage GetPrExecutors(ResourceUsage usage); 
```
I think return of ResourceUsage rather than list<ResourceUsage_Executor> would be more elegant & usefull.